packages/react/src/components/Button/Button.tsx

```diff
@@ -1,11 +1,22 @@
+import { MouseEvent } from 'react';
 import '@compassion-gds/css/src/components/Button/button.css';
 
 export interface ButtonProps {
   label: string;
+  mode?: 'primary' | 'secondary' | 'tertiary',
+  size?: 'small' | 'medium' | 'large',
+  disabled?: boolean,
+  onClick?: (event: MouseEvent<HTMLButtonElement>) => void,
 }
 
-const Button = (props: ButtonProps) => {
-  return <button className="button-main">{props.label}</button>;
+const Button = ({ label, mode = 'primary', size = 'medium', disabled = false, onClick }: ButtonProps) => {
+  const className = ['button-main', mode, size].join(' ');
```

Comment:
We probably need to allow users to pass their own css class names and styles

Can we add:
- `className?: string` to the interface
- `style?: ???` there should be a React type for this